flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/api/ExecutionConfigOptions.java

```diff
@@ -21,13 +21,24 @@
 import org.apache.flink.configuration.ConfigOption;
 import org.apache.flink.table.runtime.window.grouping.HeapWindowsGrouping;
 
+import java.time.ZoneId;
+
 import static org.apache.flink.configuration.ConfigOptions.key;
 
 /**
  * This class holds configuration constants used by Flink's table module.
  */
 public class ExecutionConfigOptions {
 
+	// ------------------------------------------------------------------------
+	//  Timestamp Options
+	// ------------------------------------------------------------------------
+	public static final ConfigOption<String> SQL_EXEC_LOCAL_TIME_ZONE_ID =
+			key("sql.exec.local.time-zone-id")
```

Comment:
Is this equivalent to TableConfig's setTimeZoneId?

Reply:
In the future, I think we need set it to configuration, for now, I'll remove it.